modules/authorization/policy-set-definitions/metadata.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
+  "name": "Policy Set Definitions",
+  "summary": "With this module you can create policy set definitions across the management group or subscription scope.",
```

Comment:
Should the README.md be updated to "This module deploys Policy Set Definitions at a Management Group or Subscription scope." and then this file updated accordingly?